Marlin/temperature.cpp

```diff
@@ -655,7 +710,12 @@ void manage_heater() {
   unsigned long ms = millis();
 
   // Loop through all extruders
-  for (int e = 0; e < EXTRUDERS; e++) {
+  #ifndef SINGLENOZZLE
+    for (int e = 0; e < EXTRUDERS; e++) 
```

Comment:
Also use `NOZZLES` here would be ok, as the loop will be removed by the compiler.